_posts/2018-12-05-multilanguage-build-system.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: posts
+title: "Bazel - a multilanguage build system"
+authors:
+  - laurentlb
+---
+
+Part of [Bazel vision](https://docs.bazel.build/versions/master/bazel-overview.html)
+is to create a multilanguage build system: it should be possible to seamlessly
+use and mix multiple languages in a codebase and to orchestrate the builds and
+the tests with a single build tool. As a codebase gets bigger, this feature
+becomes more and more important.
+
+When a user wants a build system to be multilanguage, they might mean:
+
+*   Is the build system extensible? In other words, can I add support for a new
+    language, even if it doesn't have much adoption yet, or if it is internal?
+*   How many languages are already supported?
+*   How big is the ecosystem?
+*   Does it provide a high level of quality for the languages I use?
+*   Can I use it in production?
+
+Bazel allows you to [create your own rules](https://docs.bazel.build/versions/master/skylark/concepts.html).
+More importantly, however, it allows rules to communicate and pass information
+using "providers." For example, if a rule depends on a C++ library, it might
```

Comment:
nit: switch ." to ".

Reply:
@spomorski didn't like it.
It's common in the US to put the punctuation inside quotes (but I'm not a fan of this convention).